NeoML/src/Dnn/Layers/ChannelwiseWith1x1Layer.cpp

```diff
@@ -145,31 +145,55 @@ void CChannelwiseWith1x1Layer::Reshape()
 	}
 	outputDescs[0].SetDimSize( BD_Channels, outputChannels );
 
+	recreateConvDesc( inputChannels );
+	recreateRowwiseDesc();
+}
+
+void CChannelwiseWith1x1Layer::recreateConvDesc( int inputChannels )
```

Comment:
_minor_ you don't need parameter here.
`recreateConvDesc` already uses  `inputDescs[0]`.
`const int inputChannels = inputDescs[0].Channels()`

Reply:
fixed